### PR TITLE
IA-3073

### PR DESCRIPTION
**Pull request by mshiyk**

### CODE REVIEW

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 name: Demo
 on:
   push:
+    branches:
```
Comment (jsmigovsky) [inline .github/workflows/dev.yml:5]: branches-ignore

File: .github/workflows/prod.yml [hunk 2]
```diff
@@ -15,6 +15,10 @@ permissions:
   contents: read # contents is used for checking out code
   id-token: write
 
+env:
```
Comment (jsmigovsky) [inline .github/workflows/prod.yml:18]: Please add the branches-ignore to at the top of the file

File: .github/workflows/stg.yml [hunk 3]
```diff
@@ -15,6 +15,10 @@ permissions:
   contents: read # contents is used for checking out code
   id-token: write
 
+env:
```
Comment (jsmigovsky) [inline .github/workflows/stg.yml:18]: please add branches-ignore to the top of the file

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -12,6 +12,10 @@ permissions:
   contents: read # contents is used for checking out code
   id-token: write
 
+env:
```
Comment (jsmigovsky) [inline .github/workflows/tests.yml:15]: please add tags-ignore at the top of the file
please add branches at the top of the file

File: Gemfile [hunk 5]
```diff
@@ -15,9 +15,9 @@ gem 'rails', '~> 6.1.7.3' # Bundle edge Rails instead: gem 'rails', github: 'rai
 gem 'rb-readline' # Why is this suddenly such a problem?
 gem 'sass-rails', '~> 5.0' # Use SCSS for stylesheets
 gem 'secure_headers'
-gem 'sprockets', '>=3.7.2','<4'
-gem 'sprockets-rails', :require => 'sprockets/railtie'
+gem 'sprockets', '>=3.7.2', '<4'
 gem 'sprockets-es6'
+gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'strong_migrations', '~> 0.3' # Catch unsafe migrations at dev time
 gem 'uglifier', '>= 1.3.0' # Use Uglifier as compressor for JavaScript assets
```
Comment (jsmigovsky) [inline Gemfile:23]: was a change to the Gemfile necessary?

File: .github/actions/terraform/action.yml [hunk 6]
```diff
@@ -22,4 +22,4 @@ runs:
         echo "image_tag = \"${{ inputs.image }}\"" >> ./build.auto.tfvars
         terraform workspace select ${{ inputs.workspace }}
         terraform plan -out=${{ inputs.workspace }}.tfplan -input=false
-        terraform apply -input=false ${{ inputs.workspace }}.tfplan
+#        terraform apply -input=false ${{ inputs.workspace }}.tfplan
```
Comment (yaythecloud) [inline .github/actions/terraform/action.yml:25]: remove this comment

File: .github/workflows/dev.yml (hunk 1 above)
Comment (mshiyk) [reply .github/workflows/dev.yml:5]: Corrected this.

File: Gemfile (hunk 5 above)
Comment (mshiyk) [reply Gemfile:23]: Yeah it was necessary to make sure that rubocop test was passing.

File: .github/actions/terraform/action.yml (hunk 6 above)
Comment (mshiyk) [reply .github/actions/terraform/action.yml:25]: Removed the Comment.

File: .github/workflows/prod.yml (hunk 2 above)
Comment (mshiyk) [reply .github/workflows/prod.yml:18]: Added branches-ignore.

File: .github/workflows/stg.yml (hunk 3 above)
Comment (mshiyk) [reply .github/workflows/stg.yml:18]: Added branches-ignore.

File: .github/workflows/tests.yml (hunk 4 above)
Comment (mshiyk) [reply .github/workflows/tests.yml:15]: Added tags-ignore, and branches.

**Review by yaythecloud — APPROVED** [review]
Approved

**Pull request merged**